web/components/comments/comments-list.tsx

```diff
@@ -75,13 +74,21 @@ export function UserCommentsList(props: { user: User }) {
     <Col className={'bg-canvas-50'}>
       {isLoading && <LoadingIndicator className="mt-4" />}
       {!isLoading &&
-        pageComments.map(({ key, items }, i) => {
+        pageComments.map(({ items }, i) => {
+          const pinnedComments = items.filter((comment) => comment.pinned)
+          const regularComments = items.filter((comment) => !comment.pinned)
+
           return (
-            <ProfileCommentGroup
-              key={i}
-              groupKey={key}
-              items={items as ContractComment[]}
-            />
+            <div key={i}>
+              {pinnedComments.map((comment) => (
+                <ProfileComment key={comment.id} comment={comment} />
+              ))}
+
+              {/* Then render regular comments */}
+              {regularComments.map((comment) => (
+                <ProfileComment key={comment.id} comment={comment} />
+              ))}
+            </div>
```

Comment:
Isn't this a totally different meaning of pinned, if it is pinned not only on the market, but on your comments list?

Seems like you should just do without this bit.

Reply:
whoops, undid all changes to this file in my current code